server/src/main/java/org/elasticsearch/index/codec/vectors/es818/ES818BinaryQuantizedVectorsFormat.java

```diff
@@ -87,7 +88,7 @@
  */
 public class ES818BinaryQuantizedVectorsFormat extends FlatVectorsFormat {
 
-    public static final boolean USE_DIRECT_IO = Boolean.parseBoolean(System.getProperty("vector.rescoring.directio", "false"));
+    static final boolean USE_DIRECT_IO = getUseDirectIO();
```

Comment:
my bad, missed this when resolving the conflict
```suggestion
    public static final boolean USE_DIRECT_IO = getUseDirectIO();
```